lib/cc/engine/analyzers/violation.rb

```diff
@@ -48,7 +48,7 @@ def other_sexps
           @other_locations ||= sorted_hashes.drop(1)
         end
 
-        def name
+        def check_name
```

Comment:
Not sure, but I don't think so.

Issues that get better or worse don't change fingerprints. I think a duplication issue moving from Similar to Identical or back is related to, and part of, that issue getting better or worse.